runtest.py

```diff
@@ -1,10 +1,4 @@
-import unittest
 import sys
+import pytest
 
```

Comment:
I usually add this [comment][1] before the ``sys.exit()`` call. Just as a reminder.

[1]: https://github.com/palazzem/wagtail-nesting-box/blob/master/runtests.py#L6